tilelang/testing/benchmark.py

```
@@ -0,0 +1,186 @@
+
+
+def process_func(func, *args, name=None, **kwargs):
+    import torch
+
+    latency = None
+    try:
+        with suppress_output():
+            latency = func(*args, **kwargs)
+            torch.cuda.synchronize()
+    except Exception:
+        pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)